core/services/log/eth_subscriber.go

```diff
@@ -30,6 +31,8 @@ func newEthSubscriber(ethClient eth.Client, config Config, chStop chan struct{})
 	}
 }
 
+// backfillLogs - fetches earlier logs either from a relatively recent block (latest minus BlockBackfillDepth) or from the given fromBlockOverride
+// note that the whole operation has no timeout - it relies on BlockBackfillMode (set outside) to optionally prevent very deep, long backfills
```

Comment:
A little equation of worst case runtime would be useful here. I think its 1min * numBlocks/batchSize * 10 retries right. So for 10000 blocks with the default batch size of 100 worst case backfill time is 1000 minutes? Maybe we want to trim that retry down to like 3 instead of 10

Reply:
good point - changed to 3 retries and extended the comment